setup.py

```diff
@@ -6,20 +6,18 @@
 ROOT_DIR = os.path.dirname(__file__)
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
-requirements = ["nose==1.3.0", "splinter>=0.4.9", "lettuce>=0.2.19"]
+requirements = ["nose", "splinter", "lettuce"]
 try: import argparse
 except ImportError: requirements.append('argparse')
 
 setup(
     name="salad",
     description="A nice mix of great BDD ingredients",
-    author="Lieferheld QA",
-    author_email="qa@deliveryhero.com",
-    url="https://github.com/beanqueen/salad",
+    author="Jana Rekittke",
+    author_email="jana@rekittke.name ",
+    url="https://github.com/salad/salad",
     version=VERSION,
-#    download_url = ['https://github.com/gabrielfalcao/lettuce/tarball/fork', ],
     install_requires=requirements,
-#    dependency_links = ['https://github.com/gabrielfalcao/lettuce/tarball/fork#egg=lettuce-0.2.19', ],
```

Comment:
The reason for a fork of lettuce being used was that the default lettuce was destroying people's *production* databases.  I haven't followed lettuce in a couple years (and have no idea if the maintainer has listened to the thousands of yelling people), but will want to know that bug is fixed before moving to trunk lettuce.